test/cpp/api/tensor.cpp

```diff
@@ -309,12 +318,31 @@ TEST(TensorTest, TorchTensorCtorSingleDimIntegralType) {
   ASSERT_TRUE(exactly_equal(tensor[2], 3));
 }
 
-TEST(TensorTest, TorchTensorCtorSingleDimFloatingType) {
+void test_TorchTensorCtorSingleDimFloatingType_expected_dtype(c10::ScalarType default_dtype) {
+  AutoDefaultDtypeMode dtype_mode(default_dtype);
+
```

Comment:
For completeness I'd also like to see `auto tensor = torch::tensor({1.5f, 2.25f, 3.125f});`

Reply:
Added